test/remote/gateways/remote_adyen_test.rb

```diff
@@ -6,7 +6,7 @@ def setup
 
     @amount = 100
 
-    @credit_card = credit_card('3700 0000 0000 002',
```

Comment:
I'm getting a lot of test failures because of the different card number. Might need to change it back.

Reply:
I think if you want to change it to the Visa card, you'll have to also change the CVV, but I'm not sure if there was a particular reason for using AMEX as the default test credit_card or not.

Reply:
Oops, this was a merge conflict and I didn't choose the right one. Will fix this now - thanks!